### PR TITLE
Blog on setting up replica of MySQL db on GCP

### PR DESCRIPTION
**Pull request by rana-abhishek**

### CODE REVIEW

File: source/_posts/akr-replica-setup.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
+---
+title: GCP MySQL replica setup
+date: 2020-09-14 19:06:35
+author: Abhishek Rana
+category: Web
+tags:
```
Comment (Rajeev111974) [inline source/_posts/akr-replica-setup.md:6]: keyword is not mentioned in the tag place

File: source/_posts/akr-replica-setup.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
+---
```
Comment (Rajeev111974) [inline source/_posts/akr-replica-setup.md:1]: keyword is not linked with website

**Comment by Rajeev111974** [comment]
HI abhishek,

 

Some changes is required there in the blog please check.

 

With Regards

Rajeev Kumar

Operations Manager, 

Nayan India Science & Technologies (P) Ltd.

+919958083303

rajeev@nayantech.com

 

This e-mail and any attachments contain information from NAYAN Companies and are intended solely for the use of the named recipient or recipients. This e-mail may contain confidential and proprietary information. Any dissemination of this e-mail by anyone other than an intended recipient is strictly prohibited. If you are not a named recipient, you are prohibited from any further viewing of the e-mail or any attachments or from making any use of the e-mail or attachments. If you believe you have received this e-mail in error, notify the sender immediately and permanently delete the e-mail, any attachments, and all copies thereof from any drives or storage media and destroy any printouts of the e-mail or attachments.

File: source/_posts/akr-replica-setup.md (hunk 1 above)
Comment (rana-abhishek) [reply source/_posts/akr-replica-setup.md:6]: I have added keywords from your list which were relevant to blog

File: source/_posts/akr-replica-setup.md (hunk 2 above)
Comment (rana-abhishek) [reply source/_posts/akr-replica-setup.md:1]: I have added the nayan website link at the bottom/end of the blog

**Review by Rajeev111974 — APPROVED** [review]

**Comment by Rajeev111974** [comment]
HI Abhishek,

 

Below mentioned blog is ready for live, Please do the thing needful.

 

With Regards

Rajeev Kumar

Operations Manager, 

Nayan India Science & Technologies (P) Ltd.

+919958083303

rajeev@nayantech.com

 

This e-mail and any attachments contain information from NAYAN Companies and are intended solely for the use of the named recipient or recipients. This e-mail may contain confidential and proprietary information. Any dissemination of this e-mail by anyone other than an intended recipient is strictly prohibited. If you are not a named recipient, you are prohibited from any further viewing of the e-mail or any attachments or from making any use of the e-mail or attachments. If you believe you have received this e-mail in error, notify the sender immediately and permanently delete the e-mail, any attachments, and all copies thereof from any drives or storage media and destroy any printouts of the e-mail or attachments.

**Pull request merged**